auction-server/src/server.rs

```diff
@@ -265,6 +312,7 @@ pub async fn start_server(run_options: RunOptions) -> anyhow::Result<()> {
         secret_key: run_options.secret_key.clone(),
         access_tokens: RwLock::new(access_tokens),
         metrics_recorder: setup_metrics_recorder()?,
+        express_relay_idl: load_express_relay_idl()?,
```

Comment:
Is this value different for each svm chain?

Reply:
it technically will be different just in the address field. but if we keep a separate config for the program id/address by chain, then we can just reuse the same IDL, similar to how it's done in the SDK PR